players/random_player.py

```diff
@@ -0,0 +1,60 @@
+from engine.player import Player
+from engine.move import Move
+
+import random
+
+
+class RandomPlayer(Player):
+    """A real live human player"""
+
+    def __str__(self):
+        return "RandomPlayer"
+
+    def make_move(self, game_state):
+        return self.make_random_legal_move(game_state)
+
+    def make_random_legal_move(self, game_state):
+        chosen_move = random.choice(self.construct_all_legal_moves(game_state))
+        try:
+            assert chosen_move.is_playable(game_state)
+        except AssertionError:
+            print "Chosen move was unplayable: {move} ;".format(move=chosen_move)
+            print(game_state.board)
+            print(game_state.player_hands)
+            print(game_state.player_id)
+            chosen_move.is_playable(game_state)
```

Comment:
debugging. tested a couple thousand times so it's probably fine. It's possible there are some EoG edge cases I haven't considered, but I can't think of any at the moment.